README.md

```diff
@@ -165,6 +165,11 @@ To learn more and get started:
 - Attend [regular office hours and SK community events](COMMUNITY.md)
 - Follow the team on our [blog](https://aka.ms/sk/blog)
 
+## Contributor Wall of Fame
+<a href="https://github.com/microsoft/semantic-kernel/graphs/contributors">
```

Comment:
Maybe with a blank line to avoid the markdown warning, and should we consider using the markdown way instead of html tag?

```suggestion

<a href="https://github.com/microsoft/semantic-kernel/graphs/contributors">
```

for markdown, we could use 
```md
[![semantic-kernel contributors](https://contrib.rocks/image?repo=microsoft/semantic-kernel)](https://github.com/microsoft/semantic-kernel/graphs/contributors)
```

Reply:
Thanks for the suggestions, will fix in a follow up PR